docs/api-reference/account_balances.md

```diff
@@ -2,7 +2,7 @@
 sidebar_position: 3
 ---
 
-# Account Balances
+# `AccountBalance`s
```

Comment:
I'd simplify just to `AccountBalance`. I _think_ `# titles` also generate fragment links in URL, and I just `#accountbalance` works better than `#AccountBalances` (but we need to check what our doc generator actually does

Reply:
Yeah, and in the TOC for API reference I'd also expect to see `Account`, as it names a specific "class" rather than accounts in general.

Reply:
and I guess as an url, `http://localhost:3000/api-reference/account` reads better.